### ISSUE TITLE
Cannot find nuScenes blobs for converting to KITTI format in data_setup_README 

### ISSUE THREAD

**Issue by JacobKan** [abhi1kumar/DEVIANT#3]
First many thanks for your work. I am new on object detection and have spent many time on this data_setup part. Yet I cannot setup the data like your structure. 
A light suggestion: Download the [nuScenes](https://www.nuscenes.org/nuscenes#download) and [Waymo](https://waymo.com/open/download/) datasets. As my understanding, these 2 datasets can be downloaded to somewhere outside the project because we will use soft link to connect them with the DEVIANT project.

**My confusion lays in this step:**
Then follow the instructions at [convert_nuscenes_to_kitti_format_and_evaluate.sh](https://github.com/abhi1kumar/DEVIANT/blob/main/data/nusc_kitti/convert_nuscenes_to_kitti_format_and_evaluate.sh) to get nusc_kitti_org folder. 

I think I should download the datasets following yours and nuScene's github. But I found it hard to follow [convert_nuscenes_to_kitti_format_and_evaluate.sh](https://github.com/abhi1kumar/DEVIANT/blob/main/data/nusc_kitti/convert_nuscenes_to_kitti_format_and_evaluate.sh) as I don't have v1.0-trainval#number_blobs_camera.tgz and v1.0-trainval01_blobs_lidar.tgz and many other directories in this .sh file.

So I am not able to generate nusc_kitti_org  folder and continue.

**Comment by abhi1kumar** [comment]
Hi @JacobKan 

> I don't have v1.0-trainval#number_blobs_camera.tgz and v1.0-trainval01_blobs_lidar.tgz and many other directories in this .sh file.

These files are available from the [nuscenes website](https://www.nuscenes.org/nuscenes/). Here are the steps:

- Go to [nuscenes website](https://www.nuscenes.org/nuscenes/) first. Click on Downloads.
![image](https://user-images.githubusercontent.com/43099708/190446078-01d5d65f-b0d2-4dab-ad09-ec1836682006.png)

- Click on Login and enter your nuscenes account details (You may need to signup if you have not done before). After login with my nuscenes account details, I see the following screen on my end

![image](https://user-images.githubusercontent.com/43099708/190446461-28b21569-e08b-4e43-bf81-a09bf030364d.png)

- Scroll down to the **Full dataset (v1.0)** section. You should see 10 camera and 10 lidar blobs for the **Trainval** set and 1 camera and 1 lidar blob for the **Test** set.

![image](https://user-images.githubusercontent.com/43099708/190448459-fba21006-6bdf-4963-a3f4-d0bd8e99a252.png)

- Finally click on the cyan US/Asia link to download these blobs. These blobs are v1.0-trainval##_blobs_camera.tgz and v1.0-trainval##_blobs_lidar.tgz

- Once you have downloaded them,  follow the instructions at [convert_nuscenes_to_kitti_format_and_evaluate.sh](https://github.com/abhi1kumar/DEVIANT/blob/main/data/nusc_kitti/convert_nuscenes_to_kitti_format_and_evaluate.sh) and use our [patched nuscenes devkit](https://github.com/abhi1kumar/nuscenes-devkit).

Let me know if you face any issues.

**Comment by JacobKan** [comment]
Thanks for your help! With your help and deeper study I can continue now. The .sh script file is the key.

**Comment by chun-hsueh-lee** [comment]
Hi,
For the paths in [convert_nuscenes_to_kitti_format_and_evaluate.sh](https://github.com/abhi1kumar/DEVIANT/blob/main/data/nusc_kitti/convert_nuscenes_to_kitti_format_and_evaluate.sh), do I need to change the `/home/abhinavkumar` to my own $HOME path?

Can you please provide a new file with no absolute paths? This will be a lot easier!

Thank you so much!

**Comment by abhi1kumar** [comment]
> Hi, For the paths in [convert_nuscenes_to_kitti_format_and_evaluate.sh](https://github.com/abhi1kumar/DEVIANT/blob/main/data/nusc_kitti/convert_nuscenes_to_kitti_format_and_evaluate.sh), do I need to change the `/home/abhinavkumar` to my own $HOME path?

Yes, you have to replace `/home/abhinavkumar` to your own `$HOME`. 
 
> Can you please provide a new file with no absolute paths? This will be a lot easier!

The reason I did not make these paths relative is some commands require absolute path and not relative path. You could simply search for `/home/abhinavkumar` in [convert_nuscenes_to_kitti_format_and_evaluate.sh](https://github.com/abhi1kumar/DEVIANT/blob/main/data/nusc_kitti/convert_nuscenes_to_kitti_format_and_evaluate.sh) and replace them with `/home/username`.
